gui/wxpython/startup/utils.py

```diff
@@ -87,3 +87,33 @@ def create_database_directory():
         pass
 
     return None
+
+
+def get_startup_location():
+    """Check for startup location directory in distribution.
+
+    Returns startup location if found or None if nothing was found.
+    """
+    gisbase = os.getenv("GISBASE")
+    startup_location = os.path.join(gisbase, "demolocation")
+
+    # Find out if startup location exists
+    if os.path.exists(startup_location):
+        return startup_location
+    return None
+
+
+def copy_startup_location(grassdatabase, startup_location):
+    """Copy the simple startup_location with some data to GRASS database.
+    Returns True if successfully copied or False when an error was encountered.
+    """
+    src = startup_location
+    dst = os.path.join(grassdatabase, "demolocation")
+
+    # Copy source startup location into GRASS database
+    try:
+        copytree(src, dst, ignore=ignore_patterns('*.tmpl','Makefile*'))
+        return True
+    except (IOError, OSError) as why:
```

Comment:
No need to do `as why` when you are ignoring the `why` anyway.

Reply:
"as why" removed.